packages/array/src/index.ts

```diff
@@ -7,20 +7,65 @@ export type IndexedGuard<A, B extends A> = (
   index: number,
 ) => value is B
 
-export function map<A, B>(array: A[], mapfn: IndexedMap<A, B>): B[] {
+export function fill<A>(value: A, length: number): A[] {
```

Comment:
I'm wondering if it would make sense to implement `fill` as `generate(() => value, length)`

Reply:
@deiga yeah, I suppose the closure will be inlined.